spec/operators/filter-spec.ts

```diff
@@ -1,6 +1,12 @@
 import {expect} from 'chai';
 import * as Rx from '../../dist/cjs/Rx';
-declare const {hot, cold, asDiagram, expectObservable, expectSubscriptions};
+import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
```

Comment:
No, it's #1633. If use imported type, TS will transpile it and doc will be affected, reason only use it as type to be removed while transpiling.